Assets/MRTK/SDK/Profiles/DefaultMixedRealityXRSDKControllerVisualizationProfile.asset

```diff
@@ -24,9 +24,9 @@ MonoBehaviour:
     type: 3}
   globalRightControllerModel: {fileID: 1760394390528354, guid: 9ef0df44319c4fb4a05649e5b8ceb6d1,
     type: 3}
-  globalLeftHandVisualizer: {fileID: 1227372834072826, guid: 378f91dbbff9e2343b27a467467750bf,
+  globalLeftHandVisualizer: {fileID: 61451597023024772, guid: 6080fb9e3b49fda449e3d70daf6d2254,
```

Comment:
Will this change the visualizer to the rigged hand for HoloLens 2 as well? Do we want to special case the Oculus hands in the non-global settings instead?

Reply:
It does change the rigged hand on Hololens 2 when using XRSDK. Part of the reason we changed the global settings here was to have the rigged hands show up in input simulation, is there a way to have input simulation hands use their own controller visualization setting or to set aside hololens as having their own specific visualizer?